circle.yml

```diff
@@ -0,0 +1,24 @@
+---
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/github.com/resin-io/sshproxy
+    docker:
+      - image: golang:1.8.3-stretch
```

Comment:
This seems unnecessary to me when you can just run, for example, `docker run --rm -v $PWD:/go/src/github.com/resin-io/sshproxy golang:1.8.3-stretch make -C /go/src/github.com/resin-io/sshproxy test`